snapcraft/extractors/setuppy.py

```diff
@@ -56,6 +56,7 @@ def _fake_setup(*args, **kwargs):
                 raise _errors.SetupPyImportError(path=relpath, error=str(e)) from e
 
     version = params.get("version")
-    description = params.get("description")
+    summary = params.get("description")
+    description = params.get("long_description")
 
-    return ExtractedMetadata(version=version, description=description)
+    return ExtractedMetadata(version=version, summary=summary, description=description)
```

Comment:
This will require test updates (potentially new tests as well, these live under tests/unit/extractors/test_setuppy.py).